### DISCUSSION TITLE
MahApps.Metro v2.4.8 update error

### DISCUSSION THREAD

**Discussion by pkMoser** [MahApps/MahApps.Metro#4193] [Q&A] [answered]
I would like to update MahApps.Metro v2.4.8 via NuGet package, but I got this error:
"Unable to resolve dependencies. 'ControlzEx 5.0.1' is not compatible with 'MahApps.Metro 2.4.8 constraint: ControlzEx (>= 4.4.0 && < 5.0.0)'"
Is it possible to change this constraint?
Thank you for your response.

**Comment by punker76** [comment]
@pkMoser I will change this in v2.4.9 Sorry for this error.

**Comment by punker76** [comment, marked as answer]
@pkMoser v2.4.9 is out now

**Reply by pkMoser** [reply]
Thank you very much
